### DISCUSSION TITLE
Introducing Sambal - a linked data static site generator that uses schema.org json-ld as the content model

### DISCUSSION THREAD

**Discussion by chen4119** [schemaorg/schemaorg#2968] [Show and tell]
Happy to kick off the first discussion on show & tell!  Introducing Sambal, a linked data static site generator that uses schema.org json-ld as the content model.

Check out https://sambal.dev for documentation or https://github.com/chen4119/sambal for the source code.

I decided to develop this because I thought there's a lot of benefit by simply using schema.org json-ld to create content in the first place rather than transform your data to schema.org at the end.  No need to model your own schema, no need to worry about vendor lock-in, and best of all, it's SEO ready!  By natively supporting schema.org json-ld, Sambal has 3 major benefits over other static site generators.

1. It can automatically add application/ld+json, facebook, and twitter metadata tags to your webpage.

2. You can build UI themes based on the semantic meaning of your data and not rely on the property name of your data.  To illustrate the difference, it's unambiguous what blogpost tags are but you can encode tags in your blogpost with many names, i.e. keywords, categories, tags, etc.  By relying solely on property names, UI themes are brittle in nature. 

3. Leverage the power of linked data.  Unlike plain old json, json-ld can reference other data fragments with a url.  No more duplicating data in static markdown or yaml file.

Will appreciate any thoughts or comments or if you have the chance to try it out, happy to hear what is confusing or doesn't work.  Thank you!

**Comment by chrisspradling1980** [comment]
Chen:

So, based upon my general knowledge of being a fan of scheme, may I please
use an idea of relativity and ask your opinion of whether or not this
schematic is a qualified scenario?
For example, I have a site titled Independent Consulting Services, just by
the name of the site, automatically all the SEO qualified sites will be
directly naturally selected to inquire my site as the best ranked site to
contact for resolution attempt for their issue. Including the exclusion
automatically if their preference was selected to only wanting a human for
their troubleshooting request?

I'm excited to see how much all of you that have joined scheme as a project
will influence people from all over the world with complicated issues and
how much you all will level the playing field between corporate
professionals and freelance specialist.

Just a side note, the annual salary of corporate networking and information
technology specialists are going to go way down because of freelance as
well. You ladies and guys should pat yourselves on the back because you are
affecting areas of life where people rely on technology but can't talk
about their profession.

Thanks,
Chris
